### ISSUE TITLE
socket problem of python in anaconda3.

### ISSUE THREAD

**Issue by QingGo** [ContinuumIO/anaconda-issues#1285]
I have intall python3, anaconda2 and anaconda3 in my window8.1 system, but it seem that there are some problems with python3 in anaconda3 that I cant use the socket.socketpair() function. But I can use it in original python3. My computer OS is windows8.1.

```
C:\Windows\system32>C:\Users\Zeng\AppData\Local\Programs\Python\Python35\python.exe
Python 3.5.2 (v3.5.2:4def2a2901a5, Jun 25 2016, 22:18:55) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import socket
>>> s1,s2 = socket.socketpair()
>>> s1
<socket.socket fd=820, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 6087), ra
ddr=('127.0.0.1', 6088)>
>>> exit()

C:\Windows\system32>E:\Users\Zeng\Anaconda3\python.exe
Python 3.5.2 |Anaconda 4.2.0 (64-bit)| (default, Jul  5 2016, 11:41:13) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import socket
>>> s1,s2 = socket.socketpair()
(no reaction in a long time)

```
After further testing, I found that in python3 of anaconda3, I can make a socket to listen but cant not connect to a listened port. I try to uninstall my python3, anaconda2 and only reinstall anaconda3 and close my firewall and antivirus software but the problem still exists. I have try to install python3, anaconda2, and anaconda3 in another win10 computer and all of those work normally. Maybe there are something wrong with my comtuper environment.

At first what I encountered is another problem that I cant open jupyter notebook of anaconda3, which have been disscuse in another two issues(https://github.com/jupyter/notebook/issues/1936#issuecomment-264884231 & https://github.com/tornadoweb/tornado/issues/1907#issuecomment-265458175). Ather discuss and test I realized that the problem maybe came from a more fundamental socket function.

**Comment by ilanschnell** [comment]
I just tried what you did using the Anaconda Python 3.5.2 on 64-bit Windows 10, and did not run into any problems.

**Comment by takluyver** [comment]
@ilanschnell it appears to be some interaction with @QingGo's specific computer, since he was able to run the same thing successfully on another computer. I directed him here in the hope that someone might have an idea why Anaconda's Python behaves differently in this regard to Python.org Python 3.5.

**Comment by QingGo** [comment]
I found the reason! I install COMODO mouths ago and  shortly after that uninstall it. However, I never thought it would still work for my computer. Even I close my windows defender and firework, the uninstalled COMODO still prevent my computer for some specific connection. The solution way is: 
1. Open the Network Connections folder and view available connections.
2. Right-click the connection that you want to configure, and then click Properties.
3. on the General tab, under This connection uses the following items, uninstall the item include "comodo".
All the problems above were solved!
